idpi/test/test_intpl_k2p.py

```diff
@@ -0,0 +1,98 @@
+import os
+import shutil
+import subprocess
+
+import grib_decoder
+import jinja2
+import numpy as np
+import xarray as xr
+from operators.vertical_interpolation import interpolate_k2p
+
+
+def test_intpl_k2p():
+    # define target coordinates
+    tc_values = [40.0, 500.0, 600.0, 700.0, 800.0, 1100.0]
+    fx_voper_lev = "40,500,600,700,800,1100"
+    tc_units = "hPa"
+
+    # interpolation modes
+    modes = ["nearest_sfc", "linear_in_tcf", "linear_in_lntcf"]
```

Comment:
testing different modes would be neat as pytest parametrized functions: 
```
@pytest.mark.parametrize("modes", ["nearest_sfc", "linear_in_tcf", "linear_in_lntcf"])
def test_intpl_k2p(mode):
```
like that one function is called per mode, automatically by pytest
https://docs.pytest.org/en/6.2.x/parametrize.html

Reply:
Interesting. Introduced the mode parameterization now, but left the selection of mode-depoendent tolerances and translations for fieldextra in the test function.